webapp/src/shared/components/Game/Game.tsx

```diff
@@ -0,0 +1,348 @@
+import * as React from 'react';
+
+import './Game.css';
+
+// speed magnitude is px/second
+const BALL_SPEED = 400;
+const CANVAS_WIDTH = 325;
+const CANVAS_HEIGHT = 500;
+const BRICK_WIDTH = 40;
+const BRICK_HEIGHT = 5;
+const PADDLE_WIDTH = 80;
+const PADDLE_HEIGHT = 8;
+const PADDLE_SLIDE_SPEED = 300;
+const BALL_RADIUS = 8;
+const MAX_PADDLE_BOUNCE_ANGLE = (5 / 6) * Math.PI;
+const MIN_PADDLE_BOUNCE_ANGLE = Math.PI - MAX_PADDLE_BOUNCE_ANGLE;
+// const previousTimeRef = React.useRef<number | null>(null);
+// Calcular tiempo, depende de la pantalla el refresh rate
+const DELTA_TIME = 1 / 60;
+
+type GameElement = {
+  x: number;
+  y: number;
+  color: string;
+};
+
+type GamePaddle = GameElement & {
+  slide: number;
+  width: number;
+  height: number;
+};
+
+type GameBall = GameElement & {
+  vx: number;
+  vy: number;
+  radius: number;
+};
+
+type Coord = {
```

Comment:
we already have this type here webapp/src/shared/types.ts (Position, but you can rename it if you want).

Reply:
You are right, but I prefer to have this additional type since it represent a different thing. Position is used for the position in the screen and Coord is for the physics of the game inside the canvas (the game "court"), thanks!